package.json

```diff
@@ -36,8 +37,7 @@
     "flow-typed",
     "lib",
     "classes",
-    "tenants",
-    "index.js"
```

Comment:
Is leaving this out going to be a breaking change?

Reply:
no, not being used at the moment.